controller/composite/controller.go

```diff
@@ -489,7 +504,7 @@ func (pc *parentController) syncParentObject(parent *unstructured.Unstructured)
 
 	// Update parent status.
 	// We'll want to make sure this happens after manageChildren once we support observedGeneration.
-	if _, err := pc.updateParentStatus(parent, parentStatus); err != nil {
+	if _, err := pc.updateParentStatus(parent, syncResult.Status); err != nil {
```

Comment:
Will `syncResult.ParentStatus` be clearer here (to differentiate it is status of the parent, not child)?

Reply:
After the change I'm making here, the field `.Status` is now coming from the canonical [SyncHookResponse](https://sourcegraph.com/github.com/GoogleCloudPlatform/metacontroller@29995294febd79eb4d29a4eeaacc6d8f01d55524/-/blob/controller/composite/hooks.go#L39) struct, so changing the name would mean changing the public API.